src/Microsoft.TestPlatform.Utilities/Resources/Resources.resx

```diff
@@ -117,6 +117,10 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="DisplayChosenSettings" xml:space="preserve">
```

Comment:
The actual text and translated text are already available in OM resources. Possible to reuse?

Reply:
I didn't want to introduce the dependency ? Can i Delete it from there, as there's no reference to them.?

Reply:
Already there's dependency, so will remove it from UtilitiesResources.